gridstorm/models/files/rocks2.nm

```diff
@@ -1,15 +1,15 @@
 pomdp
 
 observables
-started, r1taken, r1lastobs , r2taken, r2lastobs, done
+started, r1taken, r1lastobs , r2taken, r2lastobs, done, x, y
```

Comment:
These changes break the previously used models. I would suggest creating a copy of the files where we apply these changes